examples/web/index.html

```diff
@@ -308,58 +308,42 @@ <h3 class="typography-heading-small-text">Stacked blocks</h3>
+          <p class="typography-body-large-text-bold">Body Large Text Bold</p>
+          <p class="typography-body-large-text-italic">Body Large Text Itaic</p>
+          <p class="typography-body-large-text-regular">Body Large Text Regular</p>
+          <p class="typography-body-medium-button">Body medium Button</p>
+          <p><a href="#" class="typography-body-medium-link-no-underline">Body Medium Link No Underline</a></p>
+          <p><a href="#" class="typography-body-medium-link-underline">Body Medium Link Underline</a></p>
+          <p class="typography-body-medium-text-bold">Body Medium Text Bold</p>
+          <p class="typography-body-medium-text-italic">Body Medium Text Itaic</p>
+          <p class="typography-body-medium-text-regular">Body Medium Text Regular</p>
+          <p class="typography-body-small-button">Body Small Button</p>
+          <p><a href="#" class="typography-body-small-link-no-underline">Body Small Link No Underline</a></p>
+          <p><a href="#" class="typography-body-small-link-underline">Body Small Link Underline</a></p>
+          <p class="typography-body-small-text-bold">Body Small Text Bold</p>
+          <p class="typography-body-small-text-italic">Body Small Text Itaic</p>
+          <p class="typography-body-small-text-regular">Body Small Text Regular</p>
+          <p><a href="#" class="typography-body-xsmall-link-no-underline">Body XSmall Link No Underline</a></p>
+          <p><a href="#" class="typography-body-xsmall-link-underline">Body XSmall Link Underline</a></p>
+          <p class="typography-body-xsmall-text-regular">Body XSmall Text Regular</p>
+          <p><a href="#" class="typography-heading-large-link-no-underline">Heading Large Link No Underline</a></p>
+          <p><a href="#" class="typography-heading-large-link-underline">Heading Large Link Underline</a></p>
           <p class="typography-heading-large-text">Heading Large Text</p>
-          <p><a href="#" class="typography-heading-large-link">Heading Large Link</a></p>
+          <p><a href="#" class="typography-heading-medium-link-no-underline">Heading Medium Link No Underline</a></p>
+          <p><a href="#" class="typography-heading-medium-link-underline">Heading Medium Link Underline</a></p>
           <p class="typography-heading-medium-text">Heading Medium Text</p>
-          <p><a href="#" class="typography-heading-medium-link">Heading Medium Link</a></p>
+          <p><a href="#" class="typography-heading-small-link-no-underline">Heading Small Link No Underline</a></p>
+          <p><a href="#" class="typography-heading-small-link-underline">Heading Small Link Underline</a></p>
           <p class="typography-heading-small-text">Heading Small Text</p>
-          <p><a href="#" class="typography-heading-small-link">Heading Small Link</a></p>
+          <p><a href="#" class="typography-heading-xlarge-link-no-underline">Heading XLarge Link No Underline</a></p>
```

Comment:
Could you order headings by size instead of name?

![obrazek](https://user-images.githubusercontent.com/5614085/159450437-499e9030-ef86-4e68-a4b1-de14237297e9.png)

Reply:
in examples or in exporter? Examples are easy, could be done now. Exporter will be harder as I have to check work with breakpoints.

Reply:
Just the examples.